src/spyglass/utils/dj_merge_tables.py

```diff
@@ -707,6 +707,7 @@ def source_class_dict(self) -> dict:
             self._source_class_dict = {
                 part_name: getattr(module, part_name)
                 for part_name in self.parts(camel_case=True)
+                if hasattr(module, part_name)
             }
```

Comment:
It might be worth throwing a warning about these cases to let the user know that there is code missing. 

... not that warnings do much, but it could help us debug in the future

```suggestion
            }
        for part_name in self.parts(camel_case=True):
            if part_name not in self._source_class_dict:
                logger.warning(f"Missing code for {part_name}")    
```